hawkrest/__init__.py

```diff
@@ -23,10 +23,44 @@
 default_message_expiration = 60
 
 
+def lookup_credentials(cr_id):
```

Comment:
could you rename this to `default_credentials_lookup`? Otherwise it gets overshadowed in the class method (this was why I was confused in the last review).